### PR TITLE
Updates to footer and stat edit modal

### PR DESCRIPTION
**Pull request by bqx**

### CODE REVIEW

File: src/components/ScreenMain/Footer/index.tsx [hunk 1]
```diff
@@ -15,28 +15,31 @@ const FooterContainer = styled.div`
...
-  flex-flow: row wrap;
+  flex-flow: row no-wrap;
   justify-content: center;
   gap: 8px;
+  align-items: center;
 `;
 
 const ToggleListSection = styled.div`
-  padding: 8px 8px 8px 0;
+  // padding: 8px 0 8px 8px;
```
Comment (AndaxH) [inline src/components/ScreenMain/Footer/index.tsx]: Suggestion: Commented out line can be deleted

File: src/components/ScreenMain/index.tsx [hunk 2]
```diff
@@ -95,16 +95,12 @@ const MainScreen = () => {
       <div className="nav">
         <TopNavItem className="spellbook" onClick={toggleSpellbookModal} displayName="Class Spells" />
         <TopNavItem className="talents" onClick={() => console.log("Talents Modal")} displayName="View Talents" />
-        <TopNavItem
-          className="inventory"
-          onClick={toggleScreen}
-          displayName={activeList !== "powers" ? "Powers" : "Items"}
-        />
+        <TopNavItem className="items" onClick={() => console.log("Items Modal")} displayName="Items" />
```
Comment (AndaxH) [inline src/components/ScreenMain/index.tsx:98]: 👌

**Review by AndaxH — APPROVED** [review]
Nice work Ben :D

File: src/components/ScreenMain/Footer/index.tsx (hunk 1 above)
Comment (bqx) [reply src/components/ScreenMain/Footer/index.tsx]: what kind of dumbo tries to commit commented out code?

Comment (bqx) [inline src/components/ScreenMain/Footer/index.tsx]:
```suggestion
```

**Pull request merged**